src/main/java/com/xeeva/catalog/CostCenterPage.java

```diff
@@ -1,532 +1,583 @@
+					+ "table/tbody/tr[4]/td[@class='day']"));
+			for(WebElement selWeekDate : selectDates){
+				selWeekDate.click();
+				//driver.setPageTimeout(5);
+				loopCount++;
+				break;
+			}
+			if(loopCount>0) break;
+		}
+
+		//Verify the message -'The RequiredBy date has been updated' is displayed.
+		verify_RequiredByDateUpdated_atCCLevel();
+		//driver.setElementTimeout(4);
+
+		//Get the RequiredBy date at header level.
+		String getReqByDate_AtHeaderLevel = txtReqByHeaderLevel.getText();
+		TestReporter.log("Get ReqByDate_before Change: " + getReqByDate_AtHeaderLevel);
+
+		List<WebElement> inputDates = dateCCInputLineLevel;
+		int getInputDatesSize = inputDates.size();
+		TestReporter.log("Total no. of input dates available at CC_line-level: " + getInputDatesSize);
+		//Iterate through each ReqBydate element and get text,
+		//verify the expected date with actual values.
+		for(WebElement inputDate : inputDates){
+			String getModifiedDate = inputDate.getAttribute("value");
+			TestReporter.assertEquals(getReqByDate_AtHeaderLevel, getModifiedDate, 
+					"Date has been updated at line-level for the id: - ["+inputDate.getAttribute("id")+"]");
+		}
+
+	}
+
+
+	/**
+	 * @summary Method To click button - Shop For More Items 
+	 * @author  Lalitha Banda
+	 * @date    28/09/16
+	 */
+	public void click_ShopForMoreItems(){
+		pageLoaded();
+		btnShopForMoreItems.syncVisible(5, false);
+		btnShopForMoreItems.jsClick();
+	}
+
+
+	/**
+	 * @summary Method to verify Edit link Enabled or Disabled in costcenter page
+	 * @author  Praveen Namburi
+	 * @date    27/09/16
+	 */
+	public void verifyEDITlinkEnabledOrNot(){
+		// Verify Edit for Non Price agreement 
+		List<WebElement> unitPriceEdits = driver.findElements(By.xpath(".//*[@id='customfa']/tbody/tr/td[6]/div/input"));
+		for(WebElement input : unitPriceEdits){
+			String randomID =input.getAttribute("name").replaceAll("\\D+", "");
+			String classValue = driver.findElement(By.xpath(".//*[@id='trCCRow_"+randomID+"']/td[10]/a[2]/i")).getAttribute("class");
+			if(classValue.contains("cursor-pointer")){
+				TestReporter.assertTrue(true, "Edit Item for non Price Agreement Item is Enabled !!");
+				break;
+			}
+		}
+
+		// Verify Edit for  Price agreement ,PA Expired 
+		List<WebElement> unitPriceEdits1 = driver.findElements(By.xpath(".//*[@id='customfa']/tbody/tr/td[6]/div/span"));
+		String randomID =unitPriceEdits1.get(1).getAttribute("id").replaceAll("\\D+", "");
+		String classValue = driver.findElement(By.xpath(".//*[@id='trCCRow_"+randomID+"']/td[10]/a[2]/i")).getAttribute("class");
+		if(classValue.contains("opacity")){
+			TestReporter.assertTrue(true, "Edit Item for  Price Agreement Item is Disabled !!");
+		}
+	}
+
+	/**
+	 * @summary Method to Select Shipping Address
+	 * @author  Lalitha Banda
+	 * @date    28/09/16
+	 */
+	public void selectShippingAddress(){
+		pl.isDomComplete(driver);
+		lstShippingAdd.isEnabled();
```

Comment:
If you are not doing anything with the return value of isEnabled(), then we need to remove this line.